### PR TITLE
feat(files): add files component

### PR DESCRIPTION
**Pull request by yhuard**

### CODE REVIEW

**Comment by yhuard** [comment]
Forgot to migrate https://github.com/ec-europa/digital-transformation-dev/blob/develop/lib/themes/europa/js/components/file.js

File: framework/molecules/files/files.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="file">
+  <span class="icon icon--file file__icon"></span>
```
Comment (kalinchernev) [inline framework/molecules/files/files.html]: please add the icon as dep. as well

File: framework/molecules/files/files--video.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div style="width: 50%;"> <!-- only required for the style guide -->
+  <div class="file file-video file-video--with-background">
+    <div class="file__container">
+      <iframe src="http://ec.europa.eu/avservices/play.cfm?ref=I101631" allowfullscreen="allowfullscreen" scrolling="no"></iframe>
+    </div>
+    <div class="file__caption">
+      <div class="field">
```
Comment (kalinchernev) [inline framework/molecules/files/files--video.html]: the field seems to be a dep as well

**Comment by yhuard** [comment]
Fixed

**Pull request merged**